### ISSUE TITLE
sqlite3.OperationalError: no such column: thread.recipient_ids

### ISSUE THREAD

**Issue by myanonapps** [manwefm/signal-backup#1]
I wrote a similar tool for Signal desktop but in Java.  I see the schema of the databases of desktop and Android are entirely different.  So, I attempted to use your tool and received the following.   There is no column recipient_ids in the thread table.   This is a new backup of Android Signal App and the first I attempted. 
```
Traceback (most recent call last):
  File "signal_backup.py", line 304, in <module>
    MYSELF = get_contact(db_cursor,args.your_name)
  File "signal_backup.py", line 98, in get_contact
    contact = fetch_group(db_cursor, conv_name)
  File "signal_backup.py", line 85, in fetch_group
    db_cursor.execute("SELECT groups._id, groups.title, groups.members, groups.recipient_id, thread._id FROM groups LEFT JOIN thread ON groups.recipient_id = thread.recipient_ids WHERE groups.title = ?", (group_name,))
sqlite3.OperationalError: no such column: thread.recipient_ids
```

**Comment by hjacobs** [comment]
I had the same problem and the fix is in #3

**Comment by ipfyx** [comment]
Sorry for the delay, I will look into it asap !
I am happy to see some people use my tool.
